src/components/controllers/repl/index.jsx

```diff
@@ -13,55 +14,49 @@ import REPL_CSS from './examples.css?raw';
  * @param {string} props.code
  * @param {string} [props.slug]
  */
-export function Repl({ code, slug }) {
+export function Repl({ code }) {
+	const { route } = useLocation();
+	const { query } = useRoute();
 	const [editorCode, setEditorCode] = useStoredValue('preact-www-repl-code', code);
-	const [exampleSlug, setExampleSlug] = useState(slug || '');
```

Comment:
Most of the slug changes here are, again, due to me trying to run around `iso` when I wrote this due to the awkward update order.